mealie/core/root_logger.py

```diff
@@ -9,7 +10,7 @@
 
 from .config import get_app_settings  # noqa E402
 
-LOGGER_FILE = DATA_DIR.joinpath("mealie.log")
+LOGGER_FILE = os.getenv("MEALIE_LOG_FILE", default=DATA_DIR.joinpath("mealie.log"))
```

Comment:
Is this necessary if the log file is being written to the data directory? 

I have plans for a larger logging over-hall so the env variable will get removed once that happens, and you'll have much richer control over the logging implementation

Can you kick this can down the road until we get to that refactoring? Otherwise we'd break this configuration in the next release or two.

Reply:
No worries, we'll just use this change as a patch for now, and when the refactoring is done I'll adapt the setup :slightly_smiling_face: